pep-XXXX.rst

```diff
@@ -0,0 +1,494 @@
+PEP: 9999
+Title: Callable Type Syntax
+Author: Steven Troxler <steven.troxler@gmail.com>,
+        Pradeep Kumar Srinivasan <gohanpra@gmail.com>
+Sponsor: TODO
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: xxx
+Python-Version: 3.11
+Post-History:
+
+Abstract
+========
+
+This PEP introduces a concise and friendly syntax for callable types, supporting the same functionality as ``typing.Callable`` but with an arrow syntax inspired by the syntax for typed function signatures. This allows types like ``Callable[[int, str], bool]`` to be written ``(int, str) -> bool``.
+
+The proposed syntax supports all the functionality provided by ``typing.Callable`` and ``typing.Concatenate`` and could eventually replace them entirely.
```

Comment:
Yeah I think this needs work, thanks for pointing it out.

I don't think we have consensus about whether to discuss deprecation. I'd rather not bring it up at all because that's not conventional (PEP 604 aims to deprecate Union but does not say so).

Reply:
For now I opted to go with "drop-in replacement" language, which I think makes the point we want without hinting at deprecation. We'll see what other reviewers think :)

Reply:
Nice, I think the revised wording is great!